django_cache_with_mongodb/__init__.py

```diff
@@ -218,9 +239,12 @@ def get_many(self, keys, version=None):
             }
         )
         for result in data:
-            unencoded = base64.decodebytes(result["data"])
-            unpickled = pickle.loads(unencoded)
-            out[parsed_keys[result["key"]]] = unpickled
+            if "encoded_data" in result:
```

Comment:
This should be `data`

Reply:
The `incr` method probably will break for the existing non-expired records in the cache. However, the current implementation should be faster and threadsafe. It should be a major version bump with a disclaimer.